tests/tests_luks_raid.yml

```diff
@@ -0,0 +1,135 @@
+      block:
+        - name: Create an encrypted raid volume
+          include_role:
+            name: storage
+          vars:
+            storage_volumes:
+              - name: foo
+                type: raid
+                disks: "{{ unused_disks }}"
+                raid_level: "raid0"
+                encryption: true
+                mount_point: "{{ mount_location }}"
+                size: 4g
+
+        - name: Unreachable task
+          fail:
+            msg: UNREACH
+
+      rescue:
+        - name: Check that we failed in the role
+          assert:
+            that:
+              - ansible_failed_result.msg != 'UNREACH'
+            msg: "Role has not failed when it should have"
+
+        - name: Verify the output of the keyless luks test
+          assert:
+            that: "blivet_output.failed and
+                   blivet_output.msg|regex_search('encrypted volume.*missing key') and
+                   not blivet_output.changed"
+            msg: "Unexpected behavior w/ encrypted raid pool w/ no key"
+
+    - name: Create an encrypted raid volume w/ default fs
+      include_role:
+        name: storage
+      vars:
+        storage_volumes:
+          - name: foo
+            type: raid
+            raid_level: "raid0"
+            disks: "{{ unused_disks }}"
+            encryption: true
+            encryption_password: "{{ passwd }}"
+            encryption_cipher: 'twofish-xts-plain64'
+            encryption_key_size: 512
+            encryption_luks_version: 'luks2'
+            mount_point: "{{ mount_location }}"
+            size: 4g
+
+    - include_tasks: verify-role-results.yml
+
+    - name: Remove the encryption layer
+      include_role:
+        name: storage
+      vars:
+        storage_volumes:
+          - name: foo
+            type: raid
+            raid_level: "raid0"
+            disks: "{{ unused_disks }}"
+            encryption: false
+            encryption_password: "{{ passwd }}"
+            mount_point: "{{ mount_location }}"
+            size: 4g
+
+    - include_tasks: verify-role-results.yml
+
+    - name: Generate a keyfile as password
+      copy:
+        dest: "{{ key_file }}"
+        content: 'redhat redhat'
+        
+    - name: Add encryption to the volume
+      include_role:
+        name: storage
+      vars:
+        storage_volumes:
+          - name: foo
+            type: raid
+            raid_level: "raid0"
+            disks: "{{ unused_disks }}"
+            encryption: true
+            encryption_key: "{{ key_file }}"
+            encryption_cipher: 'aes-cbc-essiv:sha256'
+            encryption_key_size: 256
+            encryption_luks_version: 'luks2'
+            mount_point: "{{ mount_location }}"
+            size: 4g
+
+    - include_tasks: verify-role-results.yml
+
+    - name: Remove keyfile
+      file:
+        path: "{{ key_file }}"
+        state: absent
+
+    - name: Clean up
+      include_role:
+        name: storage
```

Comment:
please use `name: linux-system-roles.storage`

Reply:
Hi,

have rename storage to linux-system-roles.storage, Please review

Reply:
> have rename storage to linux-system-roles.storage, Please review

lgtm